### DISCUSSION TITLE
REopt (falsely?) suggesting 100% renewable electricity generation?

### DISCUSSION THREAD

**Discussion by kmsunflower** [NREL/REopt-API-Analysis#147] [2 - Site & Utility Questions] [answered]
Hello, I am running a microgrid analysis for a site in North Carolina. I set a minimum renewable electricity target of 20%. The results suggest that during normal course of business, the facility is served by the grid and by onsite solar generation. However, it is showing that annual renewable electricity as a percent of total electricity consumption is 100%. It seems to be suggesting that the grid is 100% renewable. Am I missing something to ensure it does not register it this way? Thank you.

**Comment by adfarth** [comment, marked as answer]
Hello, currently, REopt's renewable energy accounting only accounts for the renewable energy content of on-site generation, and does not account for renewable electricity from grid-purchased electricity. By default, REopt _does_ count renewable electricity exported to the grid (e.g. from on-site solar PV) in its renewable energy calculations. I would guess that that is what is happening in your case (i.e., total solar generation, including export to the grid, equals the site load). If you would like to _not_ count exports to the grid towards the RE %, you can select "No" to the question "Count exported renewable electricity towards renewable energy goals?" under the "Renewable Energy & Emissions" accordion.
